packages/core/src/Button/index.tsx

```diff
@@ -497,3 +501,162 @@ export const IconButton = React.forwardRef<BaseElement, IconButtonProps>(
     )
   }
 )
+
+/**
+ * IconTextButton
+ *
+ * Always has a primary icon and a secondary text
+ *
+ */
+
+export const NativeIconTextButton = styled.button<{
+  readonly visibleFocus: boolean
+}>`
+  ${COMMON_STYLE}
+  border: none;
+  padding: 0 ${spacing.large} 0 0;
+  ${({ visibleFocus, theme }) => {
+    return css`
+      color: ${theme.color.text04()};
+      fill: ${theme.color.text04()};
+      background-color: transparent;
+
+      &:hover {
+        color: ${theme.color.text03()};
+        background-color: ${theme.color.element11(opacity[16])};
+        ${IconContainer} {
+          background-color: ${theme.color.textPrimary()};
+        }
+      }
+      &:focus {
+        ${visibleFocus
+          ? css`
+              color: ${theme.color.text04()};
+              background-color: ${theme.color.element11(opacity[16])};
+            `
+          : undefined};
+      }
+      &:active {
+        box-shadow: 0 0 0 4px ${theme.color.elementPrimary(opacity[24])};
+        background-color: ${theme.color.element11(opacity[24])};
+      }
+      &:disabled {
+        opacity: ${opacity[48]};
+        cursor: default;
+        box-shadow: none;
+        &:hover {
+          ${IconContainer} {
+            background-color: ${theme.color.elementPrimary()};
+          }
+        }
+      }
+    `
+  }}
+`
+const IconContainer = styled(Icon)`
+  ${({ theme }) => {
+    return css`
+      height: ${componentSize.small};
+      width: ${componentSize.small};
+      color: ${theme.color.text00()};
+      background-color: ${theme.color.elementPrimary()};
+      border-radius: ${shape.radius.small};
+      margin-right: ${spacing.medium};
+      padding: ${spacing.small};
+      transition: all 200ms;
+    `
+  }}
+`
+export interface IconTextButtonProps extends BaseProps {
```

Comment:
Instead of using `BaseProps`, you can use `BaseButtonProps`. And omit `variant` and `accent` in the `BaseButtonProps`. https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
Done.